tree/ntupleutil/v7/inc/ROOT/RNTupleImporter.hxx

```diff
@@ -90,7 +90,9 @@ Most RNTuple fields have a type identical to the corresponding TTree input branc
 Current limitations of the importer:
   - No support for trees containing TObject (or derived classes) or TClonesArray collections
   - Due to RNTuple currently storing data fully split, "don't split" markers are ignored
-  - Some types are not (yet) available in RNTuple, such as pointers, Double32_t or std::map
+  - Some types are not (yet) available in RNTuple, such as raw pointers or std::map. Please refer to the
```

Comment:
Since map is coming, perhaps like this:
```suggestion
  - Some types are not (yet) available in RNTuple, such as raw pointers. Please refer to the
```